File: Decision.java
public class Decision {
    public static void main(String[] args) {
        int nota = 72;
        if (nota >= 70) System.out.println("Aprovado com " + nota);
        else if (nota >= 40) System.out.println("Recuperação com " + nota);
        else System.out.println("Reprovado com " + nota);
    }
}

File: decision.cpp
#include <iostream>
using namespace std;
int main() {
    int nota = 72;
    if (nota >= 70) cout << "Aprovado com " << nota << endl;
    else if (nota >= 40) cout << "Recuperacao com " << nota << endl;
    else cout << "Reprovado com " << nota << endl;
    return 0;
}